### DISCUSSION TITLE
Copilot - how to accept next word of inline suggestion

### DISCUSSION THREAD

**Discussion by MatejBransky** [LazyVim/LazyVim#773] [Q&A] [answered]
Hi! Firstly, this neovim config is amazing, and I'm a happy user of it! 🎉 I highly recommend it to everyone I know. 🙂

However, I'd like to achieve a behavior where I can accept only partial suggestions from Copilot ([see this example](https://github.com/hrsh7th/nvim-cmp/discussions/1579)). 

Do you have any ideas on how to accomplish that with this config? Is it possible with the current copilot extra plugin, or should I replace it with my own setup? I've looked for related discussions and discovered that [`copilot.lua` has mappings for `accept_word` and `accept_line`](https://github.com/zbirenbaum/copilot.lua#setup-and-configuration), but I can't use them with the current `copilot_cmp` (or at least I don't know how). 

What would you suggest in this case? Copilot has been a great buddy for helping with boilerplate, and partial completions would be fantastic for that.

**Comment by folke** [comment, marked as answer]
I have no idea. This is a question for nvim-cmp and maybe for copilot-cmp, not LazyVim
